### PR TITLE
SHARE-365 Extensions as flavor

### PR DESCRIPTION
**Pull request by dmetzner**
Use extension tags to find projects similar to flavor

### CODE REVIEW

**Review by leno12 — CHANGES_REQUESTED** [review]
PR looks nice to me. Just if you can fix the failing test cases it would be great.

File: src/Catrobat/Services/TestEnv/DataFixtures/ProjectDataFixtures.php [hunk 1]
```diff
@@ -101,8 +101,8 @@ public function insertProject(array $config, bool $andFlush = true): Program
     );
     $project->setRemixMigratedAt(null);
     $project->setCatrobatVersion(1);
-    $project->setCatrobatVersionName($config['version'] ?? '');
-    $project->setLanguageVersion($config['language version'] ?? '0.92');
+    $project->setCatrobatVersionName($config['version'] ?? '0.8.5');
+    $project->setLanguageVersion($config['language version'] ?? '0.925');
```
Comment (HCrane) [inline src/Catrobat/Services/TestEnv/DataFixtures/ProjectDataFixtures.php:105]: This needs further investigation!

**Review by HCrane — APPROVED** [review]
Primarily LGTM but this Version thing needs further ivestigation...@dmetzner would you be so kind to create a research ticket?

File: src/Catrobat/Services/TestEnv/DataFixtures/ProjectDataFixtures.php (hunk 1 above)
Comment (dmetzner) [reply src/Catrobat/Services/TestEnv/DataFixtures/ProjectDataFixtures.php:105]: Done

**Pull request merged**